Macros/ACS_summary_geo_source_tr_vars.sas

```diff
@@ -486,8 +504,8 @@
     mPop18_34YearsF_&_years.="Persons 18-34 years old, Female, MOE,&_years_dash "
     mPop35_64YearsM_&_years.="Persons 35-64 years old, Male, MOE,&_years_dash"
     mPop35_64YearsF_&_years.="Persons 35-64 years old, Female, MOE,&_years_dash"
-    mPop65andOverYearsM&_years.="Persons 65 years old and over, Male, MOE,&_years_dash "
-    mPop65andOverYearsF&_years.="Persons 65 years old and over, Female, MOE,&_years_dash "
+    mPop65andOverYearsM_&_years.="Persons 65 years old and over, Male, MOE,&_years_dash "
+    mPop65andOverYearsF_&_years.="Persons 65 years old and over, Female, MOE,&_years_dash "
 
     mhshldincunder10000_&_years. = "Household income under 10000 in the past 12 months, total, MOE, &_years_dash"
 	mhshldinc10000to14999_&_years.= "Household income from 10000 to 14999 in the past 12 months, total, MOE, &_years_dash"
```

Comment:
please put these vars in the program with the rest of the ones one that topic.

Reply:
@lhendey OK, I've cleaned up ACS_summary_geo and ACS_summary_geo_source_tr_vars to put vars under their correct topics.